app.py

```diff
@@ -0,0 +1,174 @@
+import os
+from llama_stack_client import LlamaStackClient
+from tools import SANDBOX_DIR, TOOLS, run_tool
+import json
+
+MODEL_ID = "meta-llama/Llama-3.1-405B-Instruct-FP8"
```

Comment:
Using model `meta-llama/Llama-3.3-70B-Instruct` and it doesn't create anything at all.